common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -15,6 +15,24 @@ color_lists = {
 		1 = "green"
 		14 = "black"
 		0 = "purple"
+		
+		#Warcraft
+		#Exceptions
+		special_selection = {
+			trigger = {
+				NOT = {
+					gilnean_coa_trigger = yes
+					tirassian_coa_trigger = yes
+					theramoric_coa_trigger = yes
+					wastewander_coa_trigger = yes
+				}
+			}
+			30 = "red"
+			12 = "blue"
+			1 = "green"
+			14 = "black"
+			0 = "purple"
+		}
```

Comment:
It's almost what I meant except you need to remove these colors since you moved them to `special_selection`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/159189244-d32fc3f2-675b-4557-839f-8fe0739ac495.png)

</details>

Reply:
Opted to increase weights and keep fallback `normal_colours`. Ready to be tested.